profiles/Weeks-setup.yaml

```diff
@@ -274,15 +368,28 @@ gem_pouch_adjective: green
 spare_gem_pouch_container: rucksack
 
 training_abilities:
-  Tactics: 40
-  Hunt: 120
-  Perc: 120
-  App Quick: 60
+  Meraud: 3900
+#  Truffenyi: 3500
+  Pray:
+    :check: Theurgy
```

Comment:
You can remove all of the `:check:`s (and use a format similar to what you have for Meraud).